licenses/fedora.json

```diff
@@ -6548,5 +6548,155 @@
+    "url": "https://fedoraproject.org/wiki/Licensing/BSD-2-Clause-Patent"
+  },
+  "Inner Net License": {
+    "approved": "yes",
+    "fedora_abbrev": "Inner-Net",
+    "fedora_name": "Inner Net License",
+    "id": "654",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": "https://fedoraproject.org/wiki/Licensing/Inner_Net_License"
+  },
+  "NIST Software License": {
+    "approved": "yes",
+    "fedora_abbrev": "NISTSL",
+    "fedora_name": "NIST Software License",
+    "id": "655",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": "https://fedoraproject.org/wiki/Licensing/NISTSL"
+  },
+  "Perl License (variant)": {
+    "approved": "yes",
+    "fedora_abbrev": "GPLv2+ or Artistic",
+    "fedora_name": "Perl License (variant)",
+    "id": "656",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": ""
+  },
+  "Perl License (variant)": {
+    "approved": "yes",
+    "fedora_abbrev": "LGPLv2+ or Artistic",
+    "fedora_name": "Perl License (variant)",
+    "id": "657",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": ""
+  },
+  "radvd License": {
+    "approved": "yes",
+    "fedora_abbrev": "radvd",
+    "fedora_name": "radvd License",
+    "id": "658",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": "https://fedoraproject.org/wiki/Licensing/radvd_License"
+  },
+  "Tumbolia Public License": {
+    "approved": "yes",
+    "fedora_abbrev": "Tumbolia",
+    "fedora_name": "Tumbolia Public License",
+    "id": "659",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": "https://fedoraproject.org/wiki/Licensing/Tumbolia"
+  },
+  "wxWindows Library License v 3.1": {
+    "approved": "yes",
+    "fedora_abbrev": "wxWindows",
+    "fedora_name": "wxWindows Library License v 3.1",
+    "id": "660",
+    "license_text": "",
+    "spdx_abbrev": "WXwindows",
+    "spdx_name": "wxWindows Library License",
+    "url": "https://opensource.org/licenses/WXwindows"
+  },
+  "Commons Clause": {
+    "approved": "no",
+    "fedora_abbrev": "",
+    "fedora_name": "",
+    "id": "661",
+    "license_text": "",
+    "spdx_abbrev": "#No official SPDX support as of 2020-04-21",
+    "spdx_name": "#No official SPDX support as of 20202-04-21",
+    "url": "https://fedoraproject.org/wiki/Licensing/CommonsClause"
+  },
+  "Iha license": {
```

Comment:
I think this is supposed to "lha license", lowercase L.  Debian and Fedora refer to it that way and as I understand it, this is for the lha program.

Reply:
That was typo on my part, I added that one by hand.

Reply:
Do you want to fix it or have me merge it and I'll fix it up on this end?

Reply:
Updated with typo fix. You should get to sit back :-)

Reply:
Thank you, I appreciate it!